packages/es-dev-server/package.json

```diff
@@ -138,7 +138,6 @@
     "request": "^2.88.0",
     "selfsigned": "^1.10.4",
     "ts-mocha": "^7.0.0",
-    "typescript": "^3.8.3",
```

Comment:
I thought EDS was written in TS now...is that another line of work?

Reply:
@daKmoR said its not needed because its in the main package.json

Reply:
🤷